File: app/models/league.rb
class League 
  include Neo4j::ActiveNode
  property :name, type: String
  property :password, type: String

  has_one :out, :eastern_conference, model_class: Conference
  has_one :out, :western_conference, model_class: Conference
  has_many :in, :contracts, model_class: Contract, origin: :league

  validates :name, presence: true
  validates :name, uniqueness: true

  def free_players
    Player.all - contracts.map { |c| c.player }
  end

end

File: blln/app/models/league.js
import DS from 'ember-data';

export default DS.Model.extend({
  name: DS.attr("string"),
  numberOfTeams: DS.attr("number"),
  conferences: DS.hasMany("conference", {async: true}),
  sharpId: function () {
  	return "#" + this.get('id');
  }.property('sharpId'),
  contracts: DS.hasMany("contract", { async: true })
});
